### ISSUE TITLE
Suggestion: add 'tools:replace="android:theme"' to <application> element at AndroidManifest.xml:18:5-65:19 to override.

### ISSUE THREAD

**Issue by jaredsburrows** [passy/Android-DirectoryChooser#85]
When using compile 'net.rdrei.android.dirchooser:library:3.2@aar', remove the theme in the application manifest. Only use theme for the activity.

```
:processDebugManifest
/<>/AndroidManifest.xml:21:9-40 Error:
    Attribute application@theme value=(@style/AppTheme) from AndroidManifest.xml:21:9-40
    is also present at [net.rdrei.android.dirchooser:library:3.2] AndroidManifest.xml:9:18-62 value=(@style/DirectoryChooserTheme).
    Suggestion: add 'tools:replace="android:theme"' to <application> element at AndroidManifest.xml:18:5-65:19 to override.

See http://g.co/androidstudio/manifest-merger for more information about the manifest merger.

:processDebugManifest FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':processDebugManifest'.
> Manifest merger failed : Attribute application@theme value=(@style/AppTheme) from AndroidManifest.xml:21:9-40
    is also present at [net.rdrei.android.dirchooser:library:3.2] AndroidManifest.xml:9:18-62 value=(@style/DirectoryChooserTheme).
    Suggestion: add 'tools:replace="android:theme"' to <application> element at AndroidManifest.xml:18:5-65:19 to override.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 0.8 secs
```

**Comment by jaredsburrows** [comment]
This allowed me to compile: http://stackoverflow.com/a/34079182/950427.

**Comment by caoyunxuan07** [comment]
it can not work for me.
```
android:theme="@style/AppTheme"
        tools:replace="android:theme"
        tools:ignore="AllowBackup,GoogleAppIndexingWarning"
        >
```

<h3>error info</h3>
```
Error:Execution failed for task ':app:processDebugManifest'.
> Manifest merger failed : Attribute application@theme value=(@style/AppTheme) from AndroidManifest.xml:11:9
	is also present at net.rdrei.android.dirchooser:library:3.2:9:18 value=(@style/DirectoryChooserTheme)
	Suggestion: add 'tools:replace="android:theme"' to <application> element at AndroidManifest.xml:6:5 to override
	is also present at net.rdrei.android.dirchooser:library:3.2:9:18 value=(@style/DirectoryChooserTheme)
	Suggestion: add 'tools:replace="android:theme"' to <application> element at AndroidManifest.xml:6:5 to override
Error:(11, 9) Attribute application@theme value=(@style/AppTheme) from AndroidManifest.xml:11:9
D:\projectname\app\src\main\AndroidManifest.xml
```

**Comment by jaredsburrows** [comment]
Any update on this?

**Comment by sibelius** [comment]
is there any update on this one?

**Comment by ParkerK** [comment]
The latest commit fixed this. Is there an ETA when the new changes will be pushed to the repo? Thanks for all your work @passy !

**Comment by passy** [comment]
@ParkerK Do you mean in a release?

**Comment by ParkerK** [comment]
@passy yes

**Comment by pdinklag** [comment]
I see this is marked as resolved, but I just ran into this same issue.

Is this fix included in the 3.2 release, the one pointed to in the repository readme? If not, when can we expect that to happen? After all, it was applied almost a year ago.
